source/application/DbgHelp/DbgHelpStackWalkCallback.cpp

```diff
@@ -188,7 +331,8 @@ void DbgHelpStackWalkCallback::EnumerateParameters()
         m_unsure++;
         for (DWORD i = 0; i < 4; i++)
         {
-            m_param_info.emplace_back(ParamInfo{ DbgHelpArgType::Int, L"int", { static_cast<WCHAR>(L'a' + i) }, nullptr, nullptr, 4, i * 4 });
+            m_param_info.emplace_back(ParamInfo{ DbgHelpBasicType(DbgHelpBasicType::BasicType::Int32),

```

Comment:
Do we want to do this here or at printing site where it could additionally be made to work well with other arguments without names though?